diff --git a/node_modules/nestjs-dataloader/index.ts b/node_modules/nestjs-dataloader/index.ts
index addc635..750a7e6 100644
--- a/node_modules/nestjs-dataloader/index.ts
+++ b/node_modules/nestjs-dataloader/index.ts
@@ -8,9 +8,8 @@ import {
 } from '@nestjs/common';
 import { APP_INTERCEPTOR, ModuleRef, ContextIdFactory } from '@nestjs/core';
 import { GqlExecutionContext } from '@nestjs/graphql';
-import * as DataLoader from 'dataloader';
+import DataLoader from 'dataloader';
 import { Observable } from 'rxjs';
-import { idText } from 'typescript';
 
 /**
  * This interface will be used to generate the initial data loader.                
diff --git a/node_modules/nestjs-dataloader/tsconfig.json b/node_modules/nestjs-dataloader/tsconfig.json
index 6b3c242..6ec699a 100644
--- a/node_modules/nestjs-dataloader/tsconfig.json
+++ b/node_modules/nestjs-dataloader/tsconfig.json
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "esModuleInterop": true,
     "declaration": true,
     "removeComments": true,
     "emitDecoratorMetadata": true,
